pkg/clustermanager/cluster_manager.go

```diff
@@ -729,7 +731,7 @@ func (c *ClusterManager) waitForMachineDeploymentReplicasReady(ctx context.Conte
 		return nil
 	}
 
-	timeout := time.Duration(clusterSpec.Spec.WorkerNodeGroupConfigurations[0].Count) * c.machineMaxWait
```

Comment:
What count was this referring to before?

Reply:
It was referring the the count for the first worker node group only